src/Resources/doc/index.rst

```diff
@@ -43,6 +43,21 @@ optional arguments and options. Check them out with the ``--help`` option:
 
     $ php bin/console make:controller --help
 
+
+Linting Generated Code
+______________________
+
+MakerBundle uses php-cs-fixer to enforce coding standards when generating ``.php``
+files. When running a ``make`` command, MakerBundle will use a ``php-cs-fixer``
+version and configuration that is packaged with this bundle.
+
+You can explicitly set a custom path to a php-cs-fixer binary and/or configuration
+file by their respective environment variables:
+
+- ``MAKER_PHP_CS_FIXER_BINARY_PATH`` e.g. /path/to/project/php-cs-fixer.php
+- ``MAKER_PHP_CS_FIXER_CONFIG_PATH`` e.g. /path/to/project/php-cs-fixer.config.php
```

Comment:
Can I set this to just a path in my app? That'd be much more useful - e.g.

```
MAKER_PHP_CS_FIXER_CONFIG_PATH=.php-cs-fixer.dist.php
```

Same for the binary - can I just say `MAKER_PHP_CS_FIXER_BINARY_PATH=php-cs-fixer` to use a globally-installed one?

Also, I think using `.php-cs-fixer.dist.php` in the example would be more realistic.